### PR TITLE
updated step 9

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/2187

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@JohanFreelancer9-  I would like to follow up on this please.  Kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md [hunk 1]
```diff
@@ -133,7 +133,7 @@ Sign-in a domain controller or management workstations with _Domain Admin_ equiv
...
-10. In the content pane, double-click **Log on as a batch job**. Select **Define these policy settings**. and click **OK**.  Click **Add User or Group...**.  In the **Add User or Group** dialog box, click **Browse**.  In the **Select Users, Computers, Service Accounts, or Groups** dialog box, type **NT SERVICE\ALL SERVICES;DOMAINNAME\NDESSvc** where **DOMAINNAME** is the NetBios name of the domain (Example CONTOSO\NDESSvc) in **User and group names**.  Click **OK** three times.
+10. In the content pane, double-click **Log on as a service**. Select **Define these policy settings**. and click **OK**.  Click **Add User or Group...**.  In the **Add User or Group** dialog box, click **Browse**.  In the **Select Users, Computers, Service Accounts, or Groups** dialog box, type **NT SERVICE\ALL SERVICES;DOMAINNAME\NDESSvc** where **DOMAINNAME** is the NetBios name of the domain (Example CONTOSO\NDESSvc) in **User and group names**.  Click **OK** three times.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md]: In the content pane, double-click **Log on as a service**. Select **Define these policy settings** and click **OK**.  Click **Add User or Group...**.  In the **Add User or Group** dialog box, click **Browse**.  In the **Select Users, Computers, Service Accounts, or Groups** dialog box, type **NT SERVICE\ALL SERVICES;DOMAINNAME\NDESSvc** where **DOMAINNAME** is the NetBios name of the domain (Example CONTOSO\NDESSvc) in **User and group names**.  Click **OK** three times.

Comment (VLG17) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md]: @JohanFreelancer9  what do i need to change? I don't see any difference between your text and mine. Maybe i'm missing something...

Comment (JohanFreelancer9) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md]: So sorry - please just remove the full-stop.

**Define these policy settings****.** and click **OK**

Comment (VLG17) [reply windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-cert.md]: Done! Hope it's ok now

**Comment by JohanFreelancer9** [comment]
Thank you.

**Comment by mypil** [comment]
@DaniHalfin - can you please assist to check, approve and merge the content for this PR based on issue #2187?

Thank you.

**Comment by DaniHalfin** [comment]
@mapalko to confirm (thank you)

**Comment by mapalko** [comment]
@DaniHalfin looks good.

**Pull request merged**